<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>417</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Metro-helper</string>
  </property>
  <widget class="QWidget" name="central">
   <widget class="QGroupBox" name="typeBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Station Type</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="cirBtn">
       <property name="text">
        <string>Circle</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="triBtn">
       <property name="text">
        <string>Triangle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rectBtn">
       <property name="text">
        <string>Rectangle</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="setBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>163</width>
      <height>62</height>
     </rect>
    </property>
    <property name="title">
     <string>Window Setting</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="opaLab">
         <property name="text">
          <string>Opacity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="opaSlider">
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="value">
          <number>70</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="showBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Plan Railways</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startBt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBt">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Operations</string>
    </property>
    <addaction name="actionCS"/>
   </widget>
   <addaction name="menuOperation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCS">
   <property name="text">
    <string>Clean Screen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
